table/table_reader.h

```diff
@@ -99,6 +99,8 @@ class TableReader {
                          bool (*callback_func)(void* arg, const Slice& key,
                                                LazyBuffer&& value));
 
+  virtual Status ForceEvict() { return Status::NotSupported(); }
```

Comment:
Do you think we should comment on all virtual functions like others in this file? So we could understand what is the purposes of this `ForceEvict()` without reading through the callstack.

Reply:
updated